visualization/app/codeCharta/util/arrayHelper.ts

```diff
@@ -26,6 +26,29 @@ export function addItemsToArray<T>(array: T[], items: T[]) {
     return newArray
 }
 
+export function compareContent<T>(array1: T[], array2: T[]): boolean {
+    if (array1.length !== array2.length) {
+        return false
+    }
+
+    let clonedArray2 = [...array2]
+
+    return array1.every(item => {
+        const index = findIndexOfItemInArray(clonedArray2, item)
+
+        if (index >= 0) {
+            clonedArray2 = removeEntryAtIndexFromArray(clonedArray2, index)
+            return true
+        }
+
+        return false
+    })
+}
```

Comment:
isn't this simpy equivalent dequal(array1, array2)? at least that's what I understood from the docs
https://www.npmjs.com/package/dequal

Reply:
It should be different since in the docu they state: "value order within Arrays does matter" and this function ignores the order of the array. Maybe we rename the function to "compareContentIgnoringOrder" to avoid that confusion?

Reply:
Aha I see yes yes that sounds good